shared/zodSchemas.ts

```diff
@@ -15,35 +15,39 @@ export const createGiftSchema = z.object({
   receiver: z.string().min(1),
 });
 
-// CHECK THIS
-// onko alla oleva rivi tarpeellinen?
-export const updateGiftSchema = z.object({
-  gift: z.string().min(1),
-  receiver: z.string().min(1),
-});
+export const updateGiftSchema = createGiftSchema;
 
 export const userSchema = z.object({
+  email: z.string().min(1).max(128).regex(emailRegex).toLowerCase(),
+  firstName: z.string().min(1).max(128),
```

Comment:
Jos `zodChecks.ts` tiedoston sisällön siirtää tänne, niin voisiko tässä käyttää siellä määritettyä kenttää niin ei tarvitsisi määritellä niitä useita kertoja?

```suggestion
  firstName: firstNameZodCheck,
```

Ja saman voisi tehdä muillekin kentille. Ehkä nimeäisin tuon myös `firstNameZodCheck` -> `firstName` tai `firstNameSchema`.

Reply:
Hyvä idea! Kiitos! :+1: